PokeAlarm/Alarms/Twilio/TwilioAlarm.py

```diff
@@ -33,15 +33,15 @@ class TwilioAlarm(Alarm):
                        "Lure will expire at <24h_time> (<time_left>)."
         },
         'gym': {
-            'message': "A Team <old_team> gym has fallen!"
-                       "It is now controlled by <new_team>. <gmaps>"
+            'message': "A Team <old_team> gym has fallen! <gmaps>"
```

Comment:
Why did you move gmaps up a line?

Reply:
I didn't. My changes in TwilioAlarm.pl only at line 83.

Reply:
Sorry about that, it was a change that was done in an earlier commit that hadn't been made to dev yet.